plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/resources/files/project_config_content

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>project_name</name>
```

Comment:
Variables to replace have to be embraced with `${ .. }` to distinguish from another text

Reply:
OK